calculators/gas_king_rating

```diff
@@ -0,0 +1,123 @@
+<?php
+
+$GLOBALS['rating_calculators']['pink_color_rating'] = [
+    'description' => '
+        gas king rating
```

Comment:
first line of a custom rating should mention author, version. Like this
`by <a href="/player/125980">Memoria</a>: per-matchup calculation v1`

Reply:
Should I simply put my name or do I have to respect this (html ?) notation ?
Because if I have to I do not know how to do it.

Reply:
yes, you should give a link to yourself. You find your profile, get the number from address bar (your id), and you are done. See the memoria and dreamless ratings.
![image](https://github.com/dima-stefantsov/dsr_ratings/assets/1370909/99250c31-e36b-4006-9b11-93b77b2a7bb3)

Reply:
Fixed ?